R/create_email_creds_file.R

```diff
@@ -111,189 +109,3 @@ create_email_creds_file <- function(user,
   # Save the credential values as an RDS file
   saveRDS(credentials, file = file)
```

Comment:
Have you considered setting permissions on the file after creating it (e.g. `Sys.chmod(mode = "0600")`)?

Also, is there a reason you made it RDS instead of something human readable like .dcf, .yml or .json?

Reply:
I could certainly modify the file permissions to make only the owner read and write. RDS, instead of human readable, was only chosen to slightly obfuscate the file contents. If human-readable is preferable, I could write out JSON. Let me know how you feel about that.

Reply:
I’d just do JSON. If they want it obfuscated they can use keyring.

Reply:
Sounds good! I will make that change soon.

Reply:
Now saved as JSON and the file has been given `0600` permissions.